systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -1328,6 +1335,148 @@ void testOwnerReferenceOfCASecrets() {
         });
     }
 
+    X509Certificate getCertificateFromSecret(Secret secret, String dataKey) {
+        String caCert = secret.getData().get(dataKey);
+        byte[] decoded = Base64.getDecoder().decode(caCert);
+        X509Certificate cacert = null;
+        try {
+            cacert = (X509Certificate)
+                    CertificateFactory.getInstance("X.509").generateCertificate(new ByteArrayInputStream(decoded));
+        } catch (CertificateException e) {
+            e.printStackTrace();
+        }
+        return cacert;
+    }
+
+    Secret getClusterSecret(String secretName) {
+        Secret secret = kubeClient().getSecret(CLUSTER_NAME_PREFIX + secretName, false);
+        if (secret == null) {
+            LOGGER.info("Systemwide secret not found. Using Cluster CAs validity {}",
+                    clusterName + "-" + CLUSTER_NAME_PREFIX + secretName);
+            secret = kubeClient().getSecret(clusterName + "-" + secretName, true);
+        }
+        LOGGER.debug("Using secret: {}", secret);
+        return secret;
+    }
```

Comment:
I wonder why you need this method. You should know in test for what secret your are looking for.

Reply:
See comment above. I will close this one as resolved, to not have duplicate conversation here.